src/Moderation/NeedsModeration.jsx

```diff
@@ -289,19 +292,16 @@ const renderItem = (item) => {
           )}
           <span style={{ marginLeft: "20px" }}>
             <Widget
-              src="${REPL_ACCOUNT}/widget/Moderation.TogglingSetButton"
+              src="${REPL_ACCOUNT}/widget/Moderation.ModerationButton"
               props={{
                 title: "Moderate Account",
                 disabled: !isModerator,
                 iconLeft: "ph-bold ph-prohibit",
                 tooltip: blockAccountHelperText + (isModerator ? "" : "\n" + disabledMessage),
                 variant: "destructive",
                 fill: "outline",
-                data: {
-                  index: {
-                    moderate: moderationDataFormat(accountId),
-                  },
-                },
+                type: pathToType("account"),
+                buildData: moderationDataFormat.bind(null, accountId, null, null),
```

Comment:
```suggestion
                buildData: moderationDataFormat(null, accountId, null, null),
```

I believe there is no need to use `.bind`

Reply:
This function gets passed in to the dialog and called there with the reason.

Reply:
Your method is written as an arrow function that's why no need to use `bind` there. Try that out 🙂